### DISCUSSION TITLE
Dark mode feature - Select the area

### DISCUSSION THREAD

**Discussion by eidamos** [WindowTop/WindowTop-App#309] [Ideas]
Hi! I recently started using your program and also its dark mode feature. It works really well even with the image detection but it would be really nice I you could set a specific area of the window/program to be inverted so it is not updating all the time because it gets laggy when the image moves and changes.
But this is just an idea anyways thank you for the program!

**Comment by gileli121** [comment]
Thanks for the feedback!

But this idea looks to me like something that most users may not use because it sounds like too much customization effort for something that should be straightforward... it is not a one-click feature. In my opinion, manually selecting an area in a window to be dark looks to me too much.
I don't know if most users will use it.

I usually try to make features to be simple as possible unless a lot of users ask for them (and it happened with the crop feature. https://github.com/WindowTop/WindowTop-App/issues/133 )
I can consider this request if more users will ask for it.

**Comment by Alzukie** [comment]
This is also a feature i am looking for, Rather than selecting areas for dark it would be wonderful to have a specific zone excluded, IE I am using it for an awful work application for designing utility poles and there are areas i cannot have inverted due to color coded wires and so forth so i am needing the center of the window not inverted, so if there were a click and drag function to create an exclusion zone (square) it would be perfect for my use case.

**Comment by gileli121** [comment]
Thanks for the feedback.
I am starting to hear more about this feature from users recently.
So, I will try to put more time into it.
